atm.js

```diff
@@ -1,21 +1,110 @@
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+//change css property of div.balance to #FF003E
+$("div.balance").css({
+    "background-color": "#FF003E",
+    "border-radius": "10px"
+    });
 
-}
+// - As a user, I want to deposit money into one of the bank accounts
+//set up event listener for click on the Deposit button
+$("#checkingAddMoney").on("click", addUpTheMoney);
+$("#savingsAddMoney").on("click", addUpTheMoney);
 
-function checkingWithdrawal(amount){
+//set up event listener for click on the Withdrawal button
+$("#withdrawFromChecking").on("click", subtractMoney);
+$("#withdrawFromSavings").on("click", subtractMoney);
 
-}
+// on "deposit" button click, execute this function
+function addUpTheMoney(){
+    //get the current balance of the account we're accessing
+    var prevBalance = $(this).siblings("div.balance");
+    var result = prevBalance.text();
+    //get the current account balance without the $
+    var currentBalance = parseInt(result.replace("$", ""));
 
-function savingsDeposit(amount){
+    //get value of the input and set as var newEntry
+    var newEntry = parseInt($(this).siblings("input.moneyInTheBank").val());
+
+    // add value of newEntry to text content of our account
+    $(prevBalance).text( function(){
+        var total = newEntry + currentBalance;
+        return "$" + total;
+    });
+    // clear the value of the input field
+    $(this).siblings("input.moneyInTheBank").val("");
+    // set the background-color of the account to grey when money is addded
+    $(this).siblings("div.balance").css("background-color", "#E3E3E3")
+}
 
+// set up a variable and a function to total up the balance of money in both accounts
+var balanceSum;
+function getSumBalances() {
+    var cBalance = $("#checkingBalance").text();
+    var checkingBalance = parseInt(cBalance.replace("$", ""));
+    var sBalance = $("#savingsBalance").text();
+    var savingsBalance = parseInt(sBalance.replace("$", ""));
+    balanceSum = savingsBalance + checkingBalance;
+    return balanceSum;
 }
+// on "withdraw" click =
+function subtractMoney() {
+    getSumBalances();
+    //get the current balance of the account we're accessing
+    var prevBalance = $(this).siblings("div.balance");
+    var result = prevBalance.text();
+    //get the current account balance without the $
+    var currentBalance = parseInt(result.replace("$", ""));
+    //get value of the input and set as var newWithdrawal
+    var newWithdrawal = parseInt($(this).siblings("input.moneyInTheBank").val());
 
-function savingsWithdrawal(amount){
+    // subtract value of html of div.balance
+    $(prevBalance).text( function(){
+        // get the new total balance
+        var total = currentBalance - newWithdrawal;
 
+        // setting up overdraft protection with and if-else statement! first we set up the savings half of the function
+        if ($(this) == $("#withdrawFromSavings")){
```

Comment:
Can you think of a way to remove the reference to one particular element on the page here?